clients/admin-ui/src/features/locations/LocationManagement.tsx

```diff
@@ -13,8 +13,7 @@ import { usePatchLocationsRegulationsMutation } from "./locations.slice";
 import { groupByContinent } from "./transformations";
 
 const SEARCH_FILTER = (location: Location, search: string) =>
-  location.name?.toLocaleLowerCase().includes(search.toLocaleLowerCase()) ||
-  location.continent?.toLocaleLowerCase().includes(search.toLocaleLowerCase());
+  location.name?.toLocaleLowerCase().includes(search.toLocaleLowerCase());
```

Comment:
I thought having the continent search was useful, but when I was playing with it I found it confusing. there were no clear reqs on search